kyuubi-server/src/main/scala/org/apache/kyuubi/engine/JpsApplicationOperation.scala

```diff
@@ -68,13 +68,20 @@ class JpsApplicationOperation extends ApplicationOperation {
         (true, s"Succeeded to terminate: $idAndCmd")
       } catch {
         case e: Exception =>
+          if (retryable && getEngine(tag).nonEmpty) {
+            killJpsApplicationByTag(tag, false)
```

Comment:
..., else

Reply:
shit.. we should return the last attempt result

Reply:
**PLEASE do follow the Code of Conduct of the Kyuubi community.*

Reply:
thanks, it is a little rush recently. will follow it.